core/controllers/blog_dashboard_test.py

```diff
@@ -561,3 +562,275 @@ def test_cannot_delete_post_by_blog_editor(self) -> None:
             expected_status_int=401)
 
         self.logout()
+
+
+class BlogDashboardBlogPostStatisticsHandlerTests(test_utils.GenericTestBase):
+
+    username = 'user'
+    user_email = 'user@example.com'
+
+    def setUp(self) -> None:
+        """Completes the sign-up process for the various users."""
+        super().setUp()
+        self.signup(
+            self.BLOG_ADMIN_EMAIL, self.BLOG_ADMIN_USERNAME)
+        self.signup(
+            self.BLOG_EDITOR_EMAIL, self.BLOG_EDITOR_USERNAME)
+        self.signup(self.user_email, self.username)
+        self.add_user_role(
+            self.BLOG_ADMIN_USERNAME, feconf.ROLE_ID_BLOG_ADMIN)
+        self.add_user_role(
+            self.BLOG_EDITOR_USERNAME,
+            feconf.ROLE_ID_BLOG_POST_EDITOR)
+        self.blog_admin_id = (
+            self.get_user_id_from_email(self.BLOG_ADMIN_EMAIL))
+        self.blog_editor_id = (
+            self.get_user_id_from_email(self.BLOG_EDITOR_EMAIL))
+
+        self.blog_post_id = 'sample_id'
+        stats_service.create_aggregated_stats_models_for_newly_published_blog_post( # pylint: disable=line-too-long
+        self.blog_post_id)
+        self.views_stats = stats_service.get_blog_post_views_stats_by_id(
+                self.blog_post_id
+            )
```

Comment:
```suggestion
        self.views_stats = stats_service.get_blog_post_views_stats_by_id(
            self.blog_post_id
        )
```
The indentation is a bit off. 
Ditto below.

Reply:
Done